po/tr.po

```diff
@@ -1137,12 +1137,12 @@ msgstr "Kaydetme iptal edildi."
 #: src/itemlistformaction.cpp:1386 src/itemviewformaction.cpp:565
 #, c-format
 msgid "Saved article to %s"
-msgstr "Yazı %s' konumuna kaydedildi"
+msgstr "Yazı, %s' konumuna kaydedildi"
 
 #: src/itemlistformaction.cpp:1389 src/itemviewformaction.cpp:569
 #, c-format
 msgid "Error: couldn't save article to %s"
-msgstr "Hata: Yazı %s' konumuna kaydedilemedi"
+msgstr "Hata: Yazı, %s' konumuna kaydedilemedi"
```

Comment:
This translation (and the one above) have only a single quote after the `%s` without an opening quote.
Is that intentional?